code/modules/hydroponics/plant_genes.dm

```diff
@@ -268,7 +268,7 @@
 	rate = 0.03
 	examine_line = "<span class='info'>It emits a soft glow.</span>"
 	trait_id = "glow"
-	var/glow_color = "#AAD84B"
+	var/glow_color = "#9ADF8B"
```

Comment:
This color has different hue/saturation values; What do #C3E381 and #CCE793 look like instead of this?
Also, you need before and after pictures of shadowshrooms, as they inherit the color(inverted) from this.